xtask/src/test_runner.rs

```diff
@@ -36,6 +37,10 @@ pub struct RunTestsArgs {
     /// Timeout for test commands in seconds
     #[clap(short, long, default_value = "15")]
     pub timeout: u64,
+
+    /// Whether to build espflash before running tests, true by default
+    #[arg(long, value_parser = clap::value_parser!(bool))]
+    pub local_espflash: Option<bool>,
```

Comment:
We can just use a `bool` and have it `false` by default, this way we dont need a value parser and we can just use it as a flag

Reply:
Then we should just use default `true`